content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,81 @@
+    * There is not a strong use case for making this configurable through the Telemetry API.
+
+When migrating Telemetry Resources to the `v1` Telemetry API, we strongly recommend removing the usage of the non-`v1` fields. Not doing this will not cause any disruptions, as Istio has chosen to maintain the same schema across API Versions to avoid unyielding API conversion issues. This is a known, complex issue with [Kubernetes API Versioning](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#overview), and the Gateway API has chosen to address this with [Release Channels](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+## Overview of Istio CRDs
+
+| API | Versions |
+|---------|------------------------|
+| **Telemetry APIs** |
+| [Telemetry](/docs/reference/config/telemetry/) |  `v1`, `v1alpha1` |
+| **Security APIs** |
+| [Authorization Policy](/docs/reference/config/security/authorization-policy/) |  `v1`, `v1beta1` |
+| [Peer Authentication](/docs/reference/config/security/peer_authentication/) |  `v1`, `v1beta1` |
+| [Request Authentication](/docs/reference/config/security/request_authentication/) |  `v1`, `v1beta1` |
+| **Networking APIs** |
+| [Destination Rule](/docs/reference/config/networking/destination-rule/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Envoy Filter](/docs/reference/config/networking/envoy-filter/) |  `v1alpha3` |
+| Istio [Gateway](/docs/reference/config/networking/gateway/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Proxy Config](/docs/reference/config/networking/proxy-config/) |  `v1beta1` |
+| [Service Entry](/docs/reference/config/networking/service-entry/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Sidecar](/docs/reference/config/networking/sidecar/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Virtual Service](/docs/reference/config/networking/virtual-service/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Workload Entry](/docs/reference/config/networking/workload-entry/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Workload Group](/docs/reference/config/networking/workload-group/) |  `v1`, `v1beta1`, `v1alpha3` |
+| **Extension APIs** |
+| [Wasm Plugin](/docs/reference/config/proxy_extensions/wasm-plugin/) |  `v1alpha1` |
+
+## Using the `v1` Istio APIs
+
+As seen above, there are a few non-`v1` APIs and fields. The Envoy Filter, Proxy Config and Wasm Plugin aren’t `v1`, and within the Telemetry API, there are 3 non-`v1` fields.
+
+Since Istio maintains the same schema across API versions, users can utilize non-`v1` fields in their `v1` Custom Resources.
+
+For risk-averse environments, we have provided a sample `Stable Profile` that can ensure only `v1` APIs and fields are used.
```

Comment:
We should expand on this a bit (and maybe even change the heading to reflect the release channels language @craigbox was suggesting). We should talk about CEL and Kubernetes versioning, and _why_ we decided to create an experimental ValidatingAdmissionPolicy (and say that this feature is only compatible with kubernetes 1.30).

Reply:
I actually kinda like the current more direct framing focused on "how do I use this" without getting into too much complex background - the "why" is interesting, but I don't think hugely relevant to most Istio end-users.

Adding a sentence addressing how platform teams can enforce or extend these policies for their app dev team customers would be useful though, and agreed we need to mention Kubernetes 1.30+ (or 1.26+ with the `ValidatingAdmissionPolicy` feature gate enabled?)

We should be careful to avoid confusion with Istio's [configuration profiles](https://istio.io/latest/docs/setup/additional-setup/config-profiles/) though, I'd suggest avoiding the term "profile" here.

```suggestion
For risk-averse environments, we have provided a sample `Stable Admission Policy` that can ensure only `v1` APIs and fields are used.
```

Reply:
There is a new Istio profile though technically. I agree we probably shouldn't advertise it though